src/components/authentication/SIgnInComponent.tsx

```diff
@@ -21,6 +21,8 @@ const SignInComponent: React.FC = () => {
             // alert("Sign-in successful!");
             localStorage.setItem("userId", response.userId);
             console.log(localStorage.getItem("userId"));
+            localStorage.setItem("isDoctor", response.isDoctor)
```

Comment:
should this be isAdmin?

Reply:
can be if you want idm

Reply:
Just thought this would technically be more clear for applying it on the pages that only 'doctors' can access but yeah it is called admin everywhere else

Reply:
I didnt change everything, just added an extra response parameter on the backend

Reply:
yeah perfect, that's what i meant. i just saw you have the branch on the backend, looks good. I'll approve this one.